install_config/cluster_metrics.adoc

```diff
@@ -194,20 +129,23 @@ After the metrics deployer runs, the output of `oc get pods` should resemble the
 ====
 
 {product-title} metrics are stored using the Cassandra database, which is
-deployed with settings of `*MAX_HEAP_SIZE=512M*` and `*NEW_HEAP_SIZE=100M*`.
-These values should cover most  {product-title} metrics installations, but you
-can modify them in the
+deployed with settings of `*openshift_metrics_cassandra_limits_memory*: 2G`; this value could
+be adjusted further based upon the avialable memory as determined by the Cassandra start script.
+This value should cover most  {product-title} metrics installations, but you
+can modify, via environment variables, `*MAX_HEAP_SIZE*` along with heap new generation
+size (`*HEAP_NEWSIZE*`) in the
 ifdef::openshift-origin[]
-link:https://github.com/openshift/origin-metrics/blob/master/cassandra/Dockerfile[Cassandra Dockerfile] 
+link:https://github.com/openshift/origin-metrics/blob/master/cassandra/Dockerfile[Cassandra Dockerfile]
 endif::openshift-origin[]
 ifdef::openshift-enterprise[]
 Cassandra Dockerfile
 endif::openshift-enterprise[]
 prior to deploying cluster metrics.
 
-By default, metrics data is stored for 7 days. You can configure this with the `*METRIC_DURATION*` parameter in the
+By default, metrics data is stored for 7 days. You can configure this by setting the `*openshift_metrics_duration*`
+variable in your hosts file which will update the upon deployment
 ifdef::openshift-origin[]
-link:https://github.com/openshift/origin-metrics/blob/master/metrics.yaml[*_metrics.yaml_* configuration file]. 
+link:https://github.com/openshift/origin-metrics/blob/master/metrics.yaml[*_metrics.yaml_* configuration file].
 endif::openshift-origin[]
 ifdef::openshift-enterprise[]
 *_metrics.yaml_* configuration file.
```

Comment:
We should remove part about `metrics.yaml`, that was specific for the deployer pod.